File: Programs/transpose.java
// Find transpose of a matrix
import java.util.Scanner;
public class transpose
{
  public static void main(String[] args) {
    int i,j;
    Scanner sc = new Scanner(System.in);
    System.out.println("Enter size of the matrix : ");
    int n = sc.nextInt();
    int a[][] = new int[n][n];
    System.out.println("Enter the elements in the matrix : ");
    for (i=0;i<n;i++)
    {
        for (j=0;j<n;j++)
        {
            a[i][j] = sc.nextInt();
        }
    }
    int b[][] = new int[n][n];
    System.out.println("The transpose of the matrix is : ");
    for (i=0;i<n;i++)
    {
      for (j=0;j<n;j++)
      {
          b[i][j] = a[j][i];
          System.out.print(b[i][j]+" ");
      }
      System.out.println();
    }
  }
}

File: Programs/transpose.py
n = int(input('Enter the size of row matrix'))
#m = int(input('Enter the size of the column of the matrix'))
A = []
a = []
b = []
c = []
for i in range(0,n,1):
    a.append(int(input()))
for j in range(0,n,1):
    b.append(int(input()))
for k in range(0,n,1):
    c.append(int(input()))
A.append(a)
A.append(b)
A.append(c)
B =[[0,0,0],[0,0,0],[0,0,0]]
for i in range(0,n):
    for j in range(0,n):
        B[i][j] = A[j][i]
for i in range(0,len(A)):
    for j in range(0,len(A[0])):
        print(B[i][j],end=' ')
    print('\n')
